test/benchmarks/project/plugins.sbt

```diff
@@ -1,2 +1,2 @@
 addSbtPlugin("com.typesafe.sbteclipse" % "sbteclipse-plugin" % "4.0.0")
-addSbtPlugin("pl.project13.scala" % "sbt-jmh" % "0.2.16")
+addSbtPlugin("pl.project13.scala" % "sbt-jmh" % "0.2.17")
```

Comment:
Sent a [PR to update SBT-JMH](https://github.com/ktoso/sbt-jmh/pull/92), it was accepted and the new version containing the latest JMH was included